devshell/seechange_devshell.yaml

```diff
@@ -1,5 +1,24 @@
 preloads:
   - ../default_config.yaml
+overrides:
+  - local_overrides.yaml
+augments:
+  - local_augments.yaml
+
+path:
+  data_root: 'devshell/temp_data'
+  data_temp: 'devshell/temp_data'
 
 db:
-  host: seechange_postgres
\ No newline at end of file
+  host: devshell_seechange_postgres
+
+archive:
+  archive_url: http://devshell_archive:8080/
+  verify_cert: false
+  path_base: test/
+  local_read_dir: null
+  local_write_dir: null
+  token: insecure
+
+subtraction:
+  method: zogy
```

Comment:
This seems a rather specific thing to have in this config file. I think the general config already has this... I am guessing you don't actually need it...

Reply:
I copied this directly from what was in the tests config file.